saltyrtc/client.ts

```diff
@@ -36,11 +35,6 @@ export class Client {
             warning: ['connecting', 'closing', 'closed'],
             success: ['open'],
         },
-        pc: {
-            danger: ['unknown', 'init', 'new', 'failed', 'closed'],
-            warning: ['checking', 'disconnected'],
-            success: ['connected', 'completed'],
-        },
         dc: {
```

Comment:
These states are redundant to the ones defined in 35. Unless there are states in `DataChannel` that cannot be mapped to `WebSocket` states.